code/modules/admin/greyscale_modify_menu.dm

```diff
@@ -0,0 +1,97 @@
+/datum/greyscale_modify_menu
+	var/atom/target
+	var/client/user
+
+	var/list/split_colors
+
+	var/icon/preview // Eventualy this can be displayed in the ui
```

Comment:
is this a description or a todo?

Reply:
The var isn't needed anymore, the list with all the in progress icons handles the job well enough